test/README.rst

```diff
@@ -14,3 +14,24 @@ In order to run these integration tests successfully, these things which must be
 
 .. _automatically discoverable credential locations: http://boto3.readthedocs.io/en/latest/guide/configuration.html
 .. _AWS KMS CMK ARN: http://docs.aws.amazon.com/kms/latest/APIReference/API_Encrypt.html
+
+Updating Upstream Requirements
+==============================
+
+The purpose of the upstream requirements files is to provide a stable list of
+packages for pyca/cryptography's downstream test of the DynamoDB Encryption
```

Comment:
I deliberately generalized this when I was building the infrastructure, and I think this should be reflected in the description. These are for any upstream dependency that wants to test their downstream consumers. It just so happens that currently pyca/cryptography is the only instance of this.

Reply:
gotcha, fixed